src/main/resources/db/demo/16_roles.sql

```
@@ -0,0 +1,74 @@
+  'vitawear-VW109',  -- 윤태경 영업팀 과장
+  'vitawear-VW110',  -- 강민석 영업팀 사원
+  'vitawear-VW111',  -- 노현주 생산관리팀 대리
+  'vitawear-VW113'   -- 문가영 본사 대리
+);
+
+-- MASTER 1
+UPDATE account SET role='MASTER' WHERE user_id='vitawear-VW112';  -- 배수진 본사 과장
+
+-- ── 화면 접근 권한 ────────────────────────────────────────────────
+-- ADMIN 6명에게 관리자 콘솔·재무를, MEMBER 6명에게 프로젝트 권한을 준다.
+DELETE FROM page_permission WHERE user_id LIKE 'vitawear-%';
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)